### ISSUE TITLE
Post-Ghost: Highlighting text

### ISSUE THREAD

**Issue by davidwolinsky** [giles-v/ghost-theme-dont-die#2]
An option to tick on each interview page that enables readers to have passages with the most emphasis be highlighted in some fashion. Something beyond simple itals / bold, which will help them standout against the formatting present elsewhere. 

Potential vision board touchstones:

https://www.theverge.com/a/luka-artificial-intelligence-memorial-roman-mazurenko-bot
http://sprzny.uniqlo.com/uk/products/shrigley/

Side note: May pair well with the goal of someday including audio passages for readers to hear.

**Comment by davidwolinsky** [comment]
If we can make it clear it is just *my* emphasis as the shepherd of the project, the person on the phone, the person managing all conversations around all the conversations that go on the site, I think it might be helpful. A concession to the fact that these interviews ARE walls of texts and people might appreciate the extra emphasis (mine) being added. 

It's hard to measure, so maybe you have an alternate recommendation, but there's always certain passages, parts, or explorations that jump out to me when I hear and re-read these things.

I think "need" is an overstatement, but as we have 216 interviews in the can, something that helps distill might be helpful. I was reluctant about this at first, because I wanted to avoid the feeling that I am telling people what is the most important. I think if it's couched in terms of "this is worth pondering and reflecting on," especially if we bake in the ability later to HEAR the audio, it helps these interviews come alive in a different sort of way. It shouldn't feel like Cliff's Notes for Don't Die interviews, but I'm also open to different interpretations to this. I keep thinking: If people are on the site, they don't have a problem with big long posts, so why dumb it down? But it might be helpful. What do you think?

**Comment by giles-v** [comment]
@davidwolinsky this is another 2-year-old issue which isn't clear as a task to me reading it today, both in regards to your current priorities and the _a priori_ value proposition. In part, what this sounds like is a pull quote; but it also has crossovers with how Medium maintains "top highlights" on the page, which I assume is measured when someone selects any text (on the assumption that they're copying that text to a tweet or something). 

The latter idea is interesting, although your pieces are quite long as you state.

**Comment by giles-v** [comment]
@davidwolinsky let me be more direct with a question: is this a priority now? I would venture probably not and what we want to do is more significant changes to discovery and so forth. Thoughts?

**Comment by davidwolinsky** [comment]
I agree with you - not a priority right now. I have some thoughts on how we
could interpret this through where we are steering the site now, but would
rather we get the big pieces in place and then we can see about things like
this. Let's brush this aside and put a pin in it for now!
